Src/BootCamp.Chapter/Program.cs

```diff
@@ -23,10 +23,35 @@ static void Main(string[] args)
 
             Console.Write("Hello world!" + System.Environment.NewLine);
 
-            Console.WriteLine("Tom's age is " + tAge + "and his height is " + tHeight + " and his weight is " + tWeight + " BMI is: " + tBmi);
-            Console.WriteLine("Colby's age is " + cAge + "and his height is " + cHeight + " and his weight is " + cWeight + " BMI is: " + cBmi);
+            Console.WriteLine("Tom's age is " + tAge + " and his height is " + tHeight + " and his weight is " + tWeight + " BMI is: " + tBmi);
+            Console.WriteLine("Colby's age is " + cAge + " and his height is " + cHeight + " and his weight is " + cWeight + " BMI is: " + cBmi);
             Console.WriteLine("Blakes's age is " + bAge + " and his height is " + bHeight + " and his weight is " + bWeight + " BMI is: " + bBmi);
+
+            Console.WriteLine("Press any key to continue...");
+
+            Console.ReadLine();
+            Console.Clear();
+
+        Start:
+
+            Console.WriteLine("Please enter your name. ");
+            string yourName = Console.ReadLine();
+            Console.WriteLine("Hello " + yourName);
+
+            Console.WriteLine("Please enter your age now. ");
+            int yourAge = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Please enter your height now. ");
+            double yourHeight = double.Parse(Console.ReadLine());
+            Console.WriteLine("Please enter your weight now. ");
+            int yourWeight = Convert.ToInt32(Console.ReadLine());
+            double yourBMI = yourWeight / yourHeight;
+            Console.WriteLine("Alright " + yourName + ", " + "you are " + yourAge + " years old, and your BMI based off your provided measurements is " + yourBMI);
```

Comment:
Note the spec for this assignment would have you ask for name and surname.

Reply:
Alright I reworked it to ask for surname and to also not use goto